### ISSUE TITLE
Remove or make the energy/stamina system optional.

### ISSUE THREAD

**Issue by TouhouJevil** [MinicraftPlus/minicraft-plus-revived#404]
**Is your feature request related to a problem? Please describe.**
Not exactly a problem, but more one issue with the design of the game i don't like the stamina system and would prefer it got removed or got changed into something optional.
**Describe the solution you'd like**
The removal of the feature, some kind of custom game mode that allow you to disable or enable features of the game (including said stamina/energy issue) or if you want to go the simple route, have it as a toggle option in the menu or when creating a world.

**Comment by Litorom** [comment]
There is an error in this issue. I can’t find it tho.

**Comment by BenCheung0422** [comment]
The system is cool therefore I think it could be expanded (the maximum value) instead of making it optional or being removed.

**Comment by TouhouJevil** [comment]
> The system is cool therefore I think it could be expanded (the maximum value) instead of making it optional or being removed.

I don't agree the system is cool, in fact i think is bad, i would still dislike it even if it got expanded, hence me saying optional or being removed, someone made a no energy mod after i posted this issue and i had WAY more fun with the game compared to before.

**Comment by BenCheung0422** [comment]
This is related to the game balancing problem. With the stamina system, players are not allowed to interact with a high speed, so the DPS (damage per second) could be limited. If we are to reduce this system, we may put this into hardcore mode and have a new system of attack cooldown to balance it.
